content/chapter-01/LabStarAsAService.md

```diff
@@ -1,23 +1,22 @@
 # Lab: IaaS, PaaS, CaaS, FaaS, and DaaS
 
-* Do you have a account with your cloud provider?
-  * If you do, login into the cloud provider console interface
-  * If you do not have an account, create one right now. It should take from five to fifting minutes to do so
+* Loging to the cloud provider using the account you received the instructor's email
```

Comment:
Login into the cloud provider using the account info you received in the instructor's email